index.rst

```diff
@@ -19,11 +19,12 @@ for details.
    getting-started/initial-setup
 
 .. toctree::
-   :maxdepth: 1
+   :maxdepth: 2
```

Comment:
This may not need to change. This is not related the number of pages under this category.